src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx

```diff
@@ -0,0 +1,175 @@
+            {t('content.onboardingServiceProvider.headertitle')}
+          </Typography>
+          <Typography variant="body2" className="onboarding-service-desc">
+            {t('content.onboardingServiceProvider.desc')}
+          </Typography>
+          <Box
+            sx={{
+              position: 'relative',
+              height: '100%',
+              width: '90%',
+              background: '#F4FBFD',
+              margin: 'auto',
+            }}
+          >
+            <IconButton
+              aria-label="close"
+              onClick={() => {}}
+              sx={{
+                left: '0',
+                position: 'absolute',
+                top: '50%',
+                msTransform: 'translateY(-50%)',
+                transform: 'translateY(-50%)',
+              }}
+            >
+              <LinkIcon />
+            </IconButton>
+            <Box sx={{ textAlign: 'center' }}>
+              <Typography variant="body3">
+                {t('content.onboardingServiceProvider.subDesc1')}
+              </Typography>
+              <Typography variant="body3">
+                {t('content.onboardingServiceProvider.subDesc2')}
+              </Typography>
+            </Box>
+            <IconButton
+              aria-label="close"
+              onClick={() => {}}
+              sx={{
+                right: '0',
+                position: 'absolute',
+                top: '50%',
+                msTransform: 'translateY(-50%)',
+                transform: 'translateY(-50%)',
+              }}
+            >
+              <ArrowForwardIcon />
+            </IconButton>
+          </Box>
+        </div>
+      </section>
+
+      <Box>
+        <Tabs
+          value={activeTab}
+          onChange={handleTabChange}
+          sx={{ margin: '0 10% 70px' }}
+        >
+          <Tab
+            iconPosition="start"
+            icon={getTabsIcon(1)}
+            aria-controls={`simple-tabpanel-${activeTab}`}
+            id={`simple-tab-${activeTab}`}
+            label={t('content.onboardingServiceProvider.tabletitle1')}
+            sx={{
+              textTransform: 'none',
+              display: 'inline-flex',
+              width: '100%',
+              maxWidth: '550px',
+              '&.Mui-selected': {
+                borderBottom: '3px solid #0f71cb',
+              },
+            }}
+          />
+          <Tab
+            iconPosition="start"
+            icon={getTabsIcon(2)}
+            aria-controls={`simple-tabpanel-${activeTab}`}
+            id={`simple-tab-${activeTab}`}
+            label={t('content.onboardingServiceProvider.tabletitle2')}
+            sx={{
+              textTransform: 'none',
+              display: 'inline-flex',
+              width: '100%',
+              maxWidth: '550px',
+              '&.Mui-selected': {
+                borderBottom: '3px solid #0f71cb',
+              },
+            }}
+          />
+        </Tabs>
```

Comment:
again too much duplication - better create a local component with props for the differences